packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -23,7 +25,20 @@ const CFD_text = {
 // sub_account_type: "financial" | "financial_stp" | "swap_free"
 // *
 // sub_account_type financial_stp only happens in "financial" market_type
-export const getCFDAccountKey = ({ market_type, sub_account_type, platform, shortcode }) => {
+type TPlatform = 'dxtrade' | 'mt5';
```

Comment:
I'm wondering whether we should store such common types in some upper folder. Maybe we should export them and widely used in all packages, then define the same types everywhere? 
I suppose somewhere we will need them

Reply:
I'm not sure about the rest of the workspaces, but maybe we can have a file for these common types somewhere!